requests_toolbelt/downloadutils/stream.py

```diff
@@ -12,6 +12,7 @@
     r';\s*(%s|[^\s;=]+)\s*(?:=\s*(%s|[^;]+))?\s*' % (_QUOTED_STRING_RE,
                                                      _QUOTED_STRING_RE)
 )
+_DEFAULT_CHUNKSIZE = 65536
```

Comment:
Except that tee has a much different use case. In this case, we're dealing with a streamed response from requests and there have been issues in the past where we found 512 to be a good chunksize. Making this configurable will allow you to default it to whatever you'd like in your wrapper.